test/Shared/UserManagerTestBase.cs

```diff
@@ -592,8 +591,8 @@ protected TRole CreateRole(string namePrefix = "")
             {
                 IdentityResultAssert.IsSuccess(await mgr.CreateAsync(u));
             }
-            var usersQ = mgr.Users.Where(u => u.UserName.StartsWith("CanFindUsersViaUserQuerable"));
-            Assert.Null(mgr.Users.FirstOrDefault(u => u.UserName == "bogus"));
+            var usersQ = mgr.Users.Where(u => mgr.GetUserNameAsync(u).Result.StartsWith("CanFindUsersViaUserQuerable"));
+            Assert.Null(mgr.Users.FirstOrDefault(u => mgr.GetUserNameAsync(u).Result == "bogus"));
```

Comment:
For the first point, we should pass if the IQueryable is no implemented but rather skip it. I think it makes sense to have a custom xunit test attribute to skip the test. I'll prototype that but let me know if you guys think other wise